tests/models/switch_transformers/test_modeling_switch_transformers.py

```diff
@@ -576,6 +576,8 @@ class SwitchTransformersModelTest(ModelTesterMixin, GenerationTesterMixin, Pipel
     test_torchscript = False
     # The small SWITCH_TRANSFORMERS model needs higher percentages for CPU/MP tests
     model_split_percents = [0.5, 0.8, 0.9]
+    # `SwitchTransformers` is a MOE in which not all experts will get gradients
```

Comment:
```suggestion
    # `SwitchTransformers` is a MOE in which not all experts will get gradients because they are not all used in a single forward pass
```